lldb/source/Plugins/Process/Linux/NativeRegisterContextLinux_arm64.cpp

```diff
@@ -1095,13 +1114,16 @@ void NativeRegisterContextLinux_arm64::ConfigureRegisterContext() {
       m_sve_ptrace_payload.resize(SVE_PT_SIZE(vq, SVE_PT_REGS_SVE));
     }
   }
+#endif
 }
 
 uint32_t NativeRegisterContextLinux_arm64::CalculateFprOffset(
     const RegisterInfo *reg_info) const {
   return reg_info->byte_offset - GetGPRSize();
 }
 
+#if LLDB_HAVE_USER_SVE_HEADER
+xxx
```

Comment:
Should this be here?

Reply:
Thanks for catching this! This is a leftover from me trying that the conditional actually works.